src/api/subscriptions/entities/preapproval.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class Preapproval {
+  @Field(() => String)
+  id: string;
+
+  @Field(() => String)
+  status: string;
+
+  @Field(() => String)
+  reason: string;
+
+  @Field(() => String)
+  initPoint: string;
```

Comment:
what is this?

Reply:
The initPoint is the url MP gives you to redirect to the checkout page.